<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_result">
     <property name="text">
      <string>Минимальная стоимость доставки:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget_result"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushSelectDir">
     <property name="text">
      <string>browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEditDir"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushExportCSV">
     <property name="text">
      <string>export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
